server/server.js

```diff
@@ -193,8 +192,8 @@ app.get('/products/related', (req, res) => {
 // Specific and smaller queries
 // .../reviews?id=12345
 app.get('/reviews', (req, res) => {
-  const productId = req.query.id;
-  atelierQueries.getProductReviews(productId, atelierHeaders)
+  const query = req.query.product_id;
```

Comment:
nit, but I'm seeing `productId` and `product_id`, not sure if they can/should be reconciled, but something to consider